setup.py

```diff
@@ -10,7 +10,7 @@
     3,
     6,
```

Comment:
This 6 should become a 7, otherwise the assertion does not fail on 3.6.

I doubt anyone still sees this message, since we already use `python_requires` and pip won't even find a package if it does not match.  But with older pip/buildout/setuptools, I guess you can still see it.